lib/lndclient/LndClient.ts

```diff
@@ -72,6 +72,26 @@ class LndClient extends BaseClient {
     });
   }
 
+  public connect = async () => {
+    if (this.isDisabled()) {
```

Comment:
Setting status to disabled again here is unnecessary, you can just make this `if (!this.isDisabled())` and go right into the connect logic without needing an else statement.